Makefile

```diff
@@ -149,14 +149,15 @@ TIMEOUT  = 10m
 TESTPKGS = $(shell env GO111MODULE=on $(GO) list -f \
             '{{ if or .TestGoFiles .XTestGoFiles }}{{ .ImportPath }}{{ end }}' \
             $(PKGS))
-TEST_TARGETS := test-default test-bench test-short test-verbose test-race
+TEST_TARGETS := test-default test-bench test-short test-verbose test-race check-ci
 test-bench:   ARGS=-run=__absolutelynothing__ -bench=.
 test-short:   ARGS=-short
 test-verbose: ARGS=-v
 test-race:    ARGS=-race
+check-ci: 	  ENV_VARS=INTEGRATION_TESTS=true
```

Comment:
Did you intend spacing to be aligned with the ARGS stuff above? https://github.com/oslokommune/okctl/blob/1aa0da1f8af5f3568239a89229ef0089cab39086/Makefile#L157

Spacing looks a bit misaligned.